cmd/commands/runtime_install.go

```diff
@@ -1631,16 +1632,13 @@ func applySecretsToCluster(ctx context.Context, opts *RuntimeInstallOptions) err
 }
 
 func createEventsReporter(ctx context.Context, cloneOpts *apgit.CloneOptions, opts *RuntimeInstallOptions) error {
-	resPath := cloneOpts.FS.Join(apstore.Default.AppsDir, store.Get().EventsReporterName, opts.RuntimeName, "resources")
-	u, err := url.Parse(cloneOpts.URL())
+	u, err := url.Parse(cloneOpts.Repo)
```

Comment:
use the original Repo value (with sub-path and `ref` queryParam, and then `path.Join` with the `resPath`